Source/WebKit/UIProcess/API/Cocoa/APISerializedScriptValueCocoa.mm

```diff
@@ -88,20 +88,21 @@ void releaseContextIfNecessary()
 
 static SharedJSContext& sharedContext()
 {
+    ASSERT(RunLoop::isMain());
     static NeverDestroyed<SharedJSContext> sharedContext;
     return sharedContext.get();
 }
 
 id SerializedScriptValue::deserialize(WebCore::SerializedScriptValue& serializedScriptValue, JSValueRef* exception)
 {
     ASSERT(RunLoop::isMain());
-    JSContext* context = sharedContext().ensureContext();
+    RetainPtr context = sharedContext().ensureContext();
 
     JSValueRef valueRef = serializedScriptValue.deserialize([context JSGlobalContextRef], exception);
```

Comment:
Shouldn't we have a ref of the stack to the value returned by `[context JSGlobalContextRef]`?

Reply:
As long as we're hardening with smart pointers, smart point all the things! (added)